app/controllers/photos_controller.rb

```diff
@@ -16,6 +17,11 @@ def index
 
   def edit;end
 
+  def show
+    @photo = Photo.find(params[:id])
```

Comment:
It doesn't look like you're calling find_photo- you can DRY up your code a little if you call it, I think!